docs/anomaly-detection/High Availability.md

```diff
@@ -0,0 +1,245 @@
+writer:
+  datasource_url: "http://victoriametrics:8428/"
+```
+
+## Sub-configuration
+
+[Global configuration](#global-configuration) file can be split into `N` >= 1 validated **sub-configurations** based on logical entities such as [schedulers](https://docs.victoriametrics.com/anomaly-detection/components/scheduler/), [queries](https://docs.victoriametrics.com/anomaly-detection/components/models/#queries), [models](https://docs.victoriametrics.com/anomaly-detection/components/models), and [extra_filters](https://docs.victoriametrics.com/anomaly-detection/components/reader/?highlight=extra_filters#config-parameters). Each sub-configuration remains functional and respects the many-to-many relationships between models, queries, and schedulers. A minimal valid sub-configuration consists of a single [model type](https://docs.victoriametrics.com/anomaly-detection/components/models) running on a single [query](https://docs.victoriametrics.com/anomaly-detection/components/models/#queries) and attached to a single [scheduler](https://docs.victoriametrics.com/anomaly-detection/components/scheduler/).
+
+[Example config above](#configuration-example) can be split into 2 sub-configurations (by queries):
+
+- 1 model type (`zscore`), attached to 1 scheduler (`periodic_1d`) which is run on 1st query (`cpu_seconds_total`)
+- 1 model type (`zscore`), attached to 1 scheduler (`periodic_1d`) which is run on 2nd query (`host_network_receive_errors`)
+
+## Horizontal Scalability
+
+`vmanomaly` supports **horizontal scalability** {{% available_from "v1.21.0" anomaly %}} by sharding [sub-config entities](#sub-configuration), enabling workload distribution across multiple nodes while preserving consistency and maintaining a single global configuration entry point.
+
+A `vmanomaly` [global YAML configuration](#global-configuration) can be split into `N` validated [sub-configurations](#sub-configuration) based on logical entities such as [schedulers](https://docs.victoriametrics.com/anomaly-detection/components/scheduler/), [queries](https://docs.victoriametrics.com/anomaly-detection/components/models/#queries), [models](https://docs.victoriametrics.com/anomaly-detection/components/models), and [extra_filters](https://docs.victoriametrics.com/anomaly-detection/components/reader/?highlight=extra_filters#config-parameters).
+
+These [sub-configurations](#sub-configuration) can be assigned to a specific shard (node) indexed from `{0, K-1}` across `K` available nodes. 
+
+Additionally, a replication factor `R ≥ 1` ensures [high availability](#high-availability) by enforcing redundancy across shards.
+
+<p></p>
+
+<img src="../vmanomaly-sharding-ha-diagram.webp" alt="vmanomaly-sharding-ha-diagram" width="800px"/>
+
+> Please [refer to deployment options section](#deployment-options) for the examples (Docker, Docker Compose, Helm).
+
+Sharding is supported in installations via Docker or Helm and is implemented through environment variables:
```

Comment:
```suggestion
Sharding configuration can be controlled by using the following environment variables:
```
Sharding is supported with any deployment option that we support, so I would avoid pointing to some specific types.